lib/datadog/core/configuration/settings.rb

```diff
@@ -424,9 +424,29 @@ def initialize(*_)
             #
             # @default `DD_PROFILING_EXPERIMENTAL_TIMELINE_ENABLED` environment variable as a boolean, otherwise `false`
             option :experimental_timeline_enabled do |o|
+              o.after_set do |_, _, precedence|
```

Comment:
Slightly off-topic: I've noticed we don't do this in the other deprecated settings but wonder if we should:

Keep the `o.env` setting so that we warn if deprecated env variables are left lying around?

Reply:
It's an interesting idea...

Although at this point, I was thinking that for 2.0 we'd remove all these settings :thinking:. Do you think we should keep them around, potentially with the `o.env` as well?

Reply:
Nah good time to clean up house but I think it'd make sense to take this into account during the lifetime of 2.x